src/content/api/bank-account-approvals.mdx

```diff
@@ -0,0 +1,374 @@
+| :-------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| settlement      | An `approved` Bank Account Approval of type `settlement` is required for the funds in a [Settlement Wallet](https://docs.centrapay.com/api/wallets#settlement-wallets) to be released. A [Media Upload](/api/media-uploads) is uploaded by the user to provide evidence of ownership of the [Bank Account](/api/bank-accounts) to be approved. |
+| account-consent | An `approved` Bank Account Approval of type `account-consent` provides an access token to read account details from a third-party.                                                                                                                                                    |
+| payment-consent | An `approved` Bank Account Approval of type `payment-consent` provides an access token for creating payments with a third-party.                                                                                                                                                      |
+
+---
+
+## Bank Account Approval Activity Model
+
+### Attributes
+
+<Properties>
+  <Property name="activityNumber" type="number">
+    Unique sequential Bank Account Approval Activity number.
+  </Property>
+
+  <Property name="approvalId" type="string">
+    The id of the associated [Bank Account Approval](#bank-account-approval-model).
+  </Property>
+
+  <Property name="activityType" type="string">
+    The type of the Bank Account Approval Activity.
+  </Property>
+
+  <Property name="createdAt" type="timestamp">
+    When the Bank Account Approval Activity was created.
+  </Property>
+
+  <Property name="createdBy" type="crn">
+    The User or API Key that created the Bank Account Approval Activity.
+  </Property>
+</Properties>
+
+---
+
+<Endpoint
+  method="POST"
+  path="/api/bank-account-approvals"
+>
+  ## Request Bank Account Approval <Badge type="experimental"/>
+
+  This endpoint allows you to request a Bank Account Approval.
+
+  ### Attributes
+  <Properties>
+    <Property name="mediaUploadId" type="string" required>
+      The id of the associated [Media Upload][/api/media-uploads].
+    </Property>
+
+    <Property name="bankAccountId" type="string" required>
+      The id of the associated [Bank Account](/api/bank-accounts).
+    </Property>
+
+    <Property name="type" type="string" required>
+      The [Bank Account Approval Type](#bank-account-approval-types). A Bank Account Approval can only be requested for type `settlement`.
+    </Property>
+  </Properties>
+
+  ### Errors
+  <Properties>
+    <Property name="403" type="APPROVAL_ALREADY_IN_PROGRESS" error>
```

Comment:
Can you and @alexBinns please catch up on this - IMO it should be a different component or at least take different props as the property names don't really make sense 🤔

Reply:
Yeah another component makes sense - I was thinking we would just fix forward on this as the syntax will be almost identical

Reply:
Megan I've made a component for this in https://github.com/centrapay/centrapay-docs/pull/825 and have backfilled existing migrations